### ISSUE TITLE
List CLI ouput missing network

### ISSUE THREAD

**Issue by danielloader** [canonical/multipass#2871]
**Describe the bug**
Adding a bridged network to an instance isn't been shown in `multipass list`.

**To Reproduce**
1. Launch command and yaml:
```shell
multipass launch --bridged --cpus=32 --mem=32G --disk=100G --name k3s --cloud-init=cluster.yaml
```
```yaml
package_upgrade: true
runcmd:
- curl -sfL https://get.k3s.io | K3S_TOKEN=123456 INSTALL_K3S_EXEC="server" sh -
```

2. List output:
```shell
loaderd@workstation:~$ multipass list
Name                    State             IPv4             Image
k3s                     Running           10.176.120.189   Ubuntu 22.04 LTS
                                          10.42.0.0
                                          10.42.0.1
```

3. Network List output:

```shell
loaderd@workstation:~$ multipass exec k3s -- ip a show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: enp5s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 52:54:00:64:a9:8c brd ff:ff:ff:ff:ff:ff
    inet 10.176.120.189/24 metric 100 brd 10.176.120.255 scope global dynamic enp5s0
       valid_lft 2715sec preferred_lft 2715sec
    inet6 fd42:4faa:e07d:aa00:5054:ff:fe64:a98c/64 scope global dynamic mngtmpaddr noprefixroute
       valid_lft 3330sec preferred_lft 3330sec
    inet6 fe80::5054:ff:fe64:a98c/64 scope link
       valid_lft forever preferred_lft forever
3: enp6s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 52:54:00:5f:5a:11 brd ff:ff:ff:ff:ff:ff
    inet 10.0.1.137/24 metric 200 brd 10.0.1.255 scope global dynamic enp6s0
       valid_lft 85515sec preferred_lft 85515sec
    inet6 fe80::5054:ff:fe5f:5a11/64 scope link
       valid_lft forever preferred_lft forever
4: flannel.1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1450 qdisc noqueue state UNKNOWN group default
    link/ether 7e:21:48:02:ca:bd brd ff:ff:ff:ff:ff:ff
    inet 10.42.0.0/32 scope global flannel.1
       valid_lft forever preferred_lft forever
    inet6 fe80::7c21:48ff:fe02:cabd/64 scope link
       valid_lft forever preferred_lft forever
5: cni0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1450 qdisc noqueue state UP group default qlen 1000
    link/ether 8a:a0:95:15:1c:b9 brd ff:ff:ff:ff:ff:ff
    inet 10.42.0.1/24 brd 10.42.0.255 scope global cni0
       valid_lft forever preferred_lft forever
    inet6 fe80::88a0:95ff:fe15:1cb9/64 scope link
       valid_lft forever preferred_lft forever
```

**Expected behavior**
Expected `multipass list` to include the bridged IP:
```shell
loaderd@workstation:~$ multipass list
Name                    State             IPv4             Image
k3s                     Running           10.176.120.189   Ubuntu 22.04 LTS
                                          10.0.1.137
                                          10.42.0.0
                                          10.42.0.1
```

**Additional info**
 - OS: Ubuntu 22.04.1
- `multipass version` 
    multipass   1.10.1
    multipassd  1.10.1
- `multipass info --all`
    Name:           k3s
    State:          Running
    IPv4:           10.176.120.189
                    10.42.0.0
                    10.42.0.1
    Release:        Ubuntu 22.04.1 LTS
    Image hash:     e63c931f8924 (Ubuntu 22.04 LTS)
    Load:           0.00 0.07 0.14
    Disk usage:     2.6G out of 96.7G
    Memory usage:   944.3M out of 31.3G
    Mounts:         --
- `multipass get local.driver`
    lxd

**Comment by sharder996** [comment]
I'm able to reproduce this on my machine, but the issue goes away after switching to the `latest/edge` channel. Haven't been able to nail what's causing it yet.

**Comment by ricab** [comment]
Thanks for looking into this @sharder996. I didn't look closely myself, but could this be another case of #2694? That has been fixed but not released yet.

**Comment by sharder996** [comment]
I'm pretty sure this is the issue and makes sense that it is fixed in `edge`. Hey @danielloader, are you able to switch to `edge` and confirm this? 

Regardless, the fix should be coming in v1.11 which is due to be released soon.

**Comment by townsend2010** [comment]
I'm going to close this as a duplicate of #2694.
